big_tests/tests/graphql_vcard_SUITE.erl

```diff
@@ -0,0 +1,377 @@
+     ],
+     <<"title">> => [<<"TitleTest">>, <<"SEcondTest">>],
+     <<"role">> => [<<"roleTest">>, <<"SecondRole">>],
+     <<"logo">> => [<<"LogoTest">>, <<"SecondLogo">>],
+     <<"agent">> => [<<"AgentTest">>, <<"SecondAgent">>],
+     <<"org">> =>[
+         #{<<"orgname">> => <<"TESTNAME">>, <<"orgunit">> => [<<"test1">>, <<"TEST2">>]},
+         #{<<"orgname">> => <<"TESTNAME123">>, <<"orgunit">> => [<<"test1">>]}
+     ],
+     <<"categories">> => [
+         #{<<"keyword">> => [<<"KeywordTest">>]},
+         #{<<"keyword">> => [<<"KeywordTest">>, <<"Keyword2">>]}
+     ],
+     <<"note">> => [<<"NoteTest">>, <<"NOTE2">>],
+     <<"prodId">> => [<<"ProdIdTest">>, <<"PRodTEST2">>],
+     <<"rev">> => [<<"revTest">>, <<"RevTest2">>],
+     <<"sortString">> => [<<"sortStringTest">>, <<"String2">>],
+     <<"sound">> => [<<"SoundTest">>, <<"Sound2">>],
+     <<"uid">> => [<<"UidTest">>, <<"UID2">>],
+     <<"url">> => [<<"UrlTest">>, <<"URL2">>],
+     <<"desc">> => [<<"DescTest">>, <<"DESC2">>],
+     <<"class">> => [
+         #{<<"tags">> => [<<"CONFIDENTIAL">>, <<"PRIVATE">>]},
+         #{<<"tags">> => [<<"CONFIDENTIAL">>]}
+     ],
+     <<"key">> => [
+         #{<<"credential">> => <<"TESTCREDENTIAL1">>},
+         #{<<"credential">> => <<"TESTCREDENTIAL2">>}
+     ]
+    }.
+
+admin_get_address_query() ->
+    <<"query Q1($user: JID!)
+       {
+           vcard
+           {
+               getVcard(user: $user)
+               {
+                   address
+                   {
+                       tags
+                       pobox
+                       extadd
+                       street
+                       locality
+                       region
+                       pcode
+                       country
+                   }
+               }
+           }
+       }">>.
+
+user_get_query() ->
+    <<"query Q1
+       {
+           vcard
+           {
+               getVcard
+               {
+                    formattedName
+                    email
+                    {
+                        userId
+                        tags
+                    }
+               }
+           }
+       }">>.
+
+admin_get_full_vcard_as_result_mutation() ->
+    ResultFormat = get_full_vcard_as_result(),
+    <<
+       <<"mutation M1($vcard: VcardInput!, $user: JID!)">>/binary,
+       <<"{vcard{setVcard(vcard: $vcard, user: $user)">>/binary,
+       ResultFormat/binary,
+       <<"}}">>/binary
+    >>.
+
+user_get_full_vcard_as_result_mutation() ->
+    ResultFormat = get_full_vcard_as_result(),
+    <<
+       <<"mutation M1($vcard: VcardInput!){vcard{setVcard(vcard: $vcard)">>/binary,
+       ResultFormat/binary,
+       <<"}}">>/binary
+    >>.
+
+get_full_vcard_as_result() ->
+    <<"{
+           formattedName
+           nameComponents
+           {
+               family
+               givenName
+               middleName
+               prefix
+               sufix
+           }
```

Comment:
```suggestion
           { family givenName middleName prefix suffix }
```
Maybe format some of the types like this as possible? Currently, this is very long ;D It's only a proposition, for me it can be like it is now.

Reply:
I was also thinking about this, because the code layout got vertical. It's also not a must for me.